scripts/docker-compose_test.sh

```diff
@@ -8,16 +8,13 @@ fi
 
 export TRAVIS_COMMIT=${TRAVIS_COMMIT:-$(git rev-parse HEAD)}
 docker-compose build --parallel
-docker-compose -f docker-compose.yml -f docker-compose.prod.yml up -d
-trap "docker-compose down" INT EXIT
-TIMEOUT=2m
-timeout ${TIMEOUT} bash -c -- 'until [ "`docker inspect -f {{.State.Status}} $(docker-compose ps -q db)`" == "running" ]; do sleep 0.1; done;'
-timeout ${TIMEOUT} bash -c -- 'until [ "`docker inspect -f {{.State.Status}} $(docker-compose ps -q log-server)`" == "running" ]; do sleep 0.1; done;'
-timeout ${TIMEOUT} bash -c -- 'until [ "`docker inspect -f {{.State.Status}} $(docker-compose ps -q log-signer)`" == "running" ]; do sleep 0.1; done;'
-timeout ${TIMEOUT} bash -c -- 'until [ "`docker inspect -f {{.State.Status}} $(docker-compose ps -q map-server)`" == "running" ]; do sleep 0.1; done;'
-timeout ${TIMEOUT} bash -c -- 'until [ "`docker inspect -f {{.State.Health.Status}} $(docker-compose ps -q sequencer)`" == "healthy" ]; do sleep 0.1; done;'
-timeout ${TIMEOUT} bash -c -- 'until [ "`docker inspect -f {{.State.Health.Status}} $(docker-compose ps -q server)`" == "healthy" ]; do sleep 0.1; done;'
-timeout ${TIMEOUT} bash -c -- 'until [ "`docker inspect -f {{.State.Health.Status}} $(docker-compose ps -q monitor)`" == "healthy" ]; do sleep 0.1; done;'
+# Assumes there is a docker swarm already configured.
+# docker swarm init
+docker stack deploy -c docker-compose.yml -c docker-compose.prod.yml kt
```

Comment:
using the `.prod.` override in docker-compose_test.sh. is this right?

Reply:
Yes, because it was trying to test the prod config.  Not a strong opinion on this though.